.npmrc

```diff
@@ -1,2 +1,3 @@
+engine-strict=true
```

Comment:
I think I found them all, but this should help track down any I missed. It can be removed once a full build goes through.